Gemfile

```diff
@@ -1,46 +1,46 @@
-source 'https://rubygems.org'
-
-# Puma with Ruby 2.3 running on 64bit Amazon Linux/2.8.6
-ruby '2.3.8'
-
-gem 'puma'
-gem 'rake'
-
-# api
-gem 'grape'
-gem 'grape_logging'
-gem 'actionview', '>= 5.0.7.2'
-gem 'active_model_serializers'
-
-# database/orms
-gem 'pg'
-gem 'otr-activerecord'
-gem 'activerecord-postgis-adapter'
-
-# data pipeline
-gem 'aws-sdk-sns', '~> 1'
-gem 'aws-sdk-sqs', '~> 1'
-gem 'aws-sdk-s3', '~> 1'
-gem 'elasticsearch', '~> 5.0'
-
-gem 'dotenv'
-gem 'newrelic_rpm'
-
-group :development do
-  gem 'rerun'
-end
-
-group :test do
-  gem 'database_cleaner'
-  gem 'rack-test'
-  gem 'rspec'
-  gem 'rspec-json_matcher'
-  gem 'rspec_junit_formatter'
-  gem 'shoulda-matchers'
-end
-
-group :development, :test do
-  gem 'annotate'
-  gem 'factory_girl'
-  gem 'pry'
+source 'https://rubygems.org' do
```

Comment:
Placing everything in a block is apparently now a best practice: "Global source lines are a security risk and should not be used as they can lead to gems being installed from unintended sources."

https://bundler.io/gemfile.html

Reply:
novel. TIL